hedera-node/src/main/java/com/hedera/services/store/models/Token.java

```diff
@@ -234,6 +237,18 @@ public void wipe(OwnershipTracker ownershipTracker, TokenRelationship accountRel
 		setTotalSupply(newTotalSupply);
 	}
 
+    public void delete() {
```

Comment:
nit: Lets add java doc as we do for the rest of the operations such as wipe, burn etc.